### PR TITLE
Feat/st 664

### PR DESCRIPTION
**Pull request by muradalamv1**
###  https://tools.hmcts.net/jira/browse/ST-664  ###

Created a link for Contact parties in drop down

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/ContactPartiesCIC.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    @JsonProperty("RepresentativeToContact")
+    REPRESENTATIVETOCONTACT("Representative"),
+
+    @JsonProperty("ApplicantToContact")
+    APPLICANTTOCONTACT("Applicant (if different from subject)");
+
+    private final String label;
+
+    public boolean isSubject() {
+        return SUBJECTTOCONTACT.name().equalsIgnoreCase(this.name());
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/ciccase/model/ContactPartiesCIC.java:24]: For enum equality just use == ?

**Review by sjamihmcts — DISMISSED** [review]

**Review by osheaf01 — CHANGES_REQUESTED** [review]
Just one query about enum equality check?

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/ContactPartiesCIC.java (hunk 1 above)
Comment (muradalamv1) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/model/ContactPartiesCIC.java:24]: Ok. but at the moment  we do not need it ,  will change it in next ticket.

**Review by ElifDenizEsea — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/ciccase/model/ContactPartiesCIC.java (hunk 1 above)
Comment (osheaf01) [reply src/main/java/uk/gov/hmcts/sptribs/ciccase/model/ContactPartiesCIC.java:24]: Please do, it's a smell.

**Review by osheaf01 — APPROVED** [review]
LGTM bar enum equality

**Pull request merged**